Add R16 release notes

emclaug2 opened the pull request:
<!-- Include a bulleted list summarizing the main changes you have made in this PR -->
Changes proposed in this Pull Request:
- Add R16 Release Notes

Please let me know if I've missed something.

emclaug2 commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+Published our stickersheet on Figma for public viewing
+
+### Miscellaneous
```
Is this neccessary?

joebochill replied:
I don't think it hurts anything to have it in there. Most of the time we are fixing bugs and stuff, so it might be nice for people to see that we are doing maintenance on existing materials. Some other libraries like to call out specific bugs that are fixed in each release...that might be a little overkill for us.

emclaug2 replied:
Sounds good to me!  If people are ever asking about bug fix specifics I think this is something that we can integrate into github via tags instead of through our release notes.

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we have added components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
```
```suggestion
-   For Angular, we now have components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
```

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we have added components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
```
In our documentation we're referring to these as `<pxb-drawer>` etc. Do we want to do that here, or stick with the TitleCase?

emclaug2 replied:
I think for our release notes we should stick to TitleCase since that's what our earlier releases have been using and it looks fine to me.

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+-   For Angular, we have added components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
+-   For React Native, we have added a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
```
```suggestion
-   For React Native, we now have a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
```

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
+-   For React Native, we have added a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
+
+### React Native Reusable Workflow
```
```suggestion
### Reusable Workflows
```

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+### React Native Reusable Workflow
+
+Innersourced a reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native apps
```
```suggestion
We have finished innersourcing a reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native apps, allowing teams to use a common implementation across different projects.
```

emclaug2 replied:
I've used this edited version but have taken out the React Native Auth workflow link since the repo is private and this'll fail our pipeline.

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+### Design Resources
+
+Published our stickersheet on Figma for public viewing
```
```suggestion
Published our stickersheet to the Figma community
```

joebochill commented on src/docs/release-notes/R16/Summary.mdx:
```diff
@@ -0,0 +1 @@
+Added new items to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries, added search functionality to our documentation site, introduced reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native, and publicized our stickersheet in Figma.
```
```suggestion
Added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries, added search functionality to our documentation site, introduced reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native, and published our stickersheet to the Figma community.
```

huayunh commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we now have components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
```
These are pointing to the storybook dev site.

huayunh commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we now have components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
+-   For React Native, we now have a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
```
These are pointing to the storybook dev site.

emclaug2 replied:
I put these links in to put to dev site because they'll fail our CI build until our published storybook site has the new components.  An alternate solution would be to remove the links until the storybook is released.

joebochill replied:
I'll happily merge the PR in with these broken links

emclaug2 replied:
Ok I've swapped out all dev links for the master version.

huayunh commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,23 @@
+
+### Reusable Workflows
+
+We have finished innersourcing a reusable Authentication Workflow for React Native apps, allowing teams to use a common implementation across different projects.
```
Link to the NPM package page.

emclaug2 replied:
Ah good idea, I'll add a link to the NPM package.

huayunh commented:
The react monorepo is published. Do we want to mention it anywhere? Or is it not a "substantial update"?

emclaug2 commented:
> The react monorepo is published. Do we want to mention it anywhere? Or is it not a "substantial update"?

I'd def say this is substantial change to be included in the release notes.   I'll add something :)

joebochill commented:
The PX Blue CLI is also new in R16

emclaug2 commented:
> The PX Blue CLI is also new in R16

Geez I made a note to add it and still forgot; coming right up.

emclaug2 commented:
After reviewing the JIRA board for all R16 flagged items, the only other thing I can think to include in the release notes is a mention of the Typography audit/changes and theme/accessibility support for light & dark themes.  

I don't think either of these are as big as the items that are already listed so I'm leaning towards omitting them for now since our current release notes are already double the size of R15's.  If anyone wants me to add them just let me know.

joebochill commented on src/docs/release-notes/R16/R16.mdx:
```diff
@@ -0,0 +1,31 @@
+
+### PX Blue CLI
+
+Introduced the [PX Blue CLI](https://www.npmjs.com/package/@pxblue/cli), a tool used to generate new Angular, React, or React Native projects loaded with PX Blue components and themes.
```
It also does Ionic

emclaug2 replied:
Aha, updated.

huayunh commented:
Figma community DSM link: https://www.figma.com/community/file/852558784352181868

emclaug2 commented:
> Figma community DSM link: https://www.figma.com/community/file/852558784352181868

I've added this link now.

emclaug2 commented:
Ok I've removed the dev links and added in another note about RN using RN Paper & adding style overrides to our React/RN libraries.  The build will fail so it'll have to be merged manaully.

joebochill commented:
Let's hold off on merging this in just until we have the component libraries published...to keep the links working...otherwise we won't be able to deploy any other changes to doc it (if there are any more expected)
